tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "stripInternal": true,
     "noUnusedLocals": true,
     "noEmit": true,
+    "skipLibCheck": true,
```

Comment:
Needed to add this because the newer TS version complains about something in sinon types